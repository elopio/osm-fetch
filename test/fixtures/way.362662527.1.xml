<?xml version="1.0" encoding="UTF-8"?>
<osm version="0.6" generator="OpenStreetMap server" copyright="OpenStreetMap and contributors" attribution="http://www.openstreetmap.org/copyright" license="http://opendatacommons.org/licenses/odbl/1-0/">
  <way id="362662527" changeset="32919043" timestamp="2015-07-27T21:35:46Z" version="1" visible="true" user="Karta" uid="188704">
    <nd ref="3669269217"/>
    <nd ref="3669269219"/>
    <nd ref="3669269218"/>
    <nd ref="3669269220"/>
    <nd ref="3669269217"/>
    <tag k="landuse" v="farmland"/>
  </way>
</osm>
