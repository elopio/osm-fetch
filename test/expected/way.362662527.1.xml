<?xml version='1.0' encoding='UTF-8'?><osm version="0.6" generator="osm-fetch"><node id="3669269217" changeset="32919043" timestamp="2015-07-27T21:35:45Z" version="1" visible="true" user="Karta" uid="188704" lat="59.2103364" lon="11.4945311"/><node id="3669269218" changeset="32919043" timestamp="2015-07-27T21:35:46Z" version="1" visible="true" user="Karta" uid="188704" lat="59.2099629" lon="11.4953143"/><node id="3669269219" changeset="32919043" timestamp="2015-07-27T21:35:46Z" version="1" visible="true" user="Karta" uid="188704" lat="59.2101296" lon="11.4952526"/><node id="3669269220" changeset="32919043" timestamp="2015-07-27T21:35:46Z" version="1" visible="true" user="Karta" uid="188704" lat="59.2096828" lon="11.4948852"/><way id="362662527" changeset="32919043" timestamp="2015-07-27T21:35:46Z" version="1" visible="true" user="Karta" uid="188704"><nd ref="3669269217"/><nd ref="3669269219"/><nd ref="3669269218"/><nd ref="3669269220"/><nd ref="3669269217"/><tag k="landuse" v="farmland"/></way></osm>
